tests/foreman/cli/test_user.py

```diff
@@ -328,13 +329,15 @@ def test_positive_add_and_delete_roles(self, module_roles):
 class TestSshKeyInUser:
     """Implements the SSH Key in User Tests"""
 
+    ssh_key = gen_ssh_keypairs()[1]
```

Comment:
A downside to doing it this way instead of providing this via a fixture is that this function gets executed when the module is imported at collection time.

Reply:
correct. do you see an issue with this being evaluated then?